### DISCUSSION TITLE
In Animate 2021，adobe air32.0.0116 forDesktop Cannot display MP4 video

### DISCUSSION THREAD

**Discussion by daguanguan** [airsdk/Adobe-Runtime-Support#1133] [Q&A]
In Animate 2021，The external video loaded on the stage cannot be displayed in the air 32 version, but it is normal in the air 30 version
Does the air 32 version require special configuration to display MP4 video files normally

**Comment by ajwfrost** [comment]
I believe that the final/archived version of AIR 32 has had the H.264/AAC internal codecs removed due to the patent licensing requirements for these. With our latest AIR SDK (v33.1 currently) we have got platform-specific decoders being used instead in order to provide roughly the same functionality..

thanks

**Reply by daguanguan** [reply]
Oh, I see, thank you.
